pymc3/tests/test_distributions.py

```diff
@@ -19,22 +19,6 @@
 import aesara.tensor as at
 import numpy as np
 import numpy.random as nr
-
```

Comment:
I guess this removed section was a mistake?

Reply:
Oh `pre-commit` was complaining about it not being used, I though it was some sort of residual code. What does it do anyway ? (i.e. how important is it?)